### DISCUSSION TITLE
Did we forgot to cut a new release :wink: ?

### DISCUSSION THREAD

**Discussion by rsjethani** [helix-editor/helix#8201] [General]
The last release was on May 18 so more than 3 months now :)

**Comment by pascalkuthe** [comment]
Nobody forgot about it, we are waiting for an important update from an upstream library. Harassing maintainers won't make it any faster

**Reply by archseer** [reply]
There's also no timeline guarantees. We tend to aim for 2-3 months but this is on a best effort basis and we're all volunteers. 😉

**Comment by rsjethani** [comment]
thanks for the answers guys, no worries I am building from source anyways..was just reminded by a user so I thought to just ask. But appreciate the effort here. Cheers!
